.github/workflows/deploy.yml

```diff
@@ -29,16 +29,19 @@ jobs:
       run: |
         strip target/release/cbindgen
 
-    - name: Get cbindgen's version
+    - name: Handle release data and files
       id: tagName
       run: |
         VERSION=$(cargo pkgid | cut -d# -f2 | cut -d: -f2)
         echo "::set-output name=version::$VERSION"
+        tail -n +3 CHANGES | sed '/^##/q' |
```

Comment:
So this is removing the last header, then stopping at the next, and `sed '$ d'` removes the last line IIUC. Why is `awk` needed, just dedenting? Would be nice to have a comment here.

Reply:
Yep, correct. I'm going to add a comment in another PR